worker/buildbot_worker/scripts/windows_service.py

```diff
@@ -46,14 +46,14 @@
 # * If you change the buildbot configuration, you must restart the service.
 #   There is currently no way to ask a running buildbot to reload the
 #   config.  You can restart by executing:
-#   % python buildbot_service.py restart
+#   % buildbot_worker_windows_service restart
 #
 # Troubleshooting:
 # * Check the Windows event log for any errors.
 # * Check the "twistd.log" file in your buildbot directories - once each
 #   bot has been started it just writes to this log as normal.
 # * Try executing:
-#   % python buildbot_service.py debug
+#   % buildbot_windows_service debug
```

Comment:
indeed.
No idea if this is working actually. this has to be tested by windows community.